CHANGELOG.md

```diff
@@ -3,11 +3,16 @@
 # Changelog
 
 - [Changelog](#changelog)
-  - [3.10.0](#3100)
+  - [3.11.0](#3110)
```

Comment:
Hm yeah maybe "unreleased" or something.

Reply:
Yeah, unreleased/next version sounds best - mostly because sometimes we might cut a patch release (like 3.10.1 later this week) which would also include whatever's in the pipeline for "next version".

Reply:
Good idea. Changed to "Next Version"